src/pyhf/infer/test_statistics.py

```diff
@@ -1,12 +1,41 @@
 from .. import get_backend
 from .mle import fixed_poi_fit, fit
-from ..exceptions import UnspecifiedPOI
+from ..exceptions import UnspecifiedPOI, InvalidTestStatistic
 
 import logging
 
 log = logging.getLogger(__name__)
 
 
+def get(name):
+    """
+    Get the test statistic function by name.
+
+    Example:
+
+        >>> import pyhf
+        >>> pyhf.infer.test_statistics.get("q")
+        <function qmu at 0x...>
+        >>> pyhf.infer.test_statistics.get("qtilde")
```

Comment:
Only applies if people agree on the API comment
```suggestion
        >>> pyhf.infer.test_statistics.get_test_stat("q")
        <function qmu at 0x...>
        >>> pyhf.infer.test_statistics.get_test_stat("qtilde")
```